certbot-dns-cloudxns/certbot_dns_cloudxns/_internal/dns_cloudxns.py

```diff
@@ -78,10 +82,12 @@ def __init__(self, api_key, secret_key, ttl):
 
         self.provider = cloudxns.Provider(config)
 
-    def _handle_http_error(self, e, domain_name):
+    def _handle_http_error(self, e: HTTPError, domain_name: str) -> errors.PluginError:
```

Comment:
Well, for the returned type, it is acceptable to return a subtype of the overriden method one since any call assuming the super class will manipulate a reference that is valid by all means.

However it is not my intention to artificially limit the possibilities in the overriden methods, so I fix that.